scripts/main.py

```diff
@@ -0,0 +1,150 @@
+import os
+from dotenv import load_dotenv
+import tweepy as tw
+import re
+import string
+import nltk
+import pandas as pd
+from sklearn.cluster import DBSCAN
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+# NLTK variables to aid in cleaning/preprocessing
+ps = nltk.PorterStemmer()
+
+# Pandas dataframe for tweets
+df = pd.DataFrame(columns=["id", "tweet", "popularity"])
+
+
+class Results:
+    def __init__(self):
+        """Initialize final results of the analysis"""
+        self.clusters_count = df.clusters.value_counts()
+        self.df_results = df.groupby(["clusters"]).max().reset_index()
+        print("Number of tweets per cluster: \n{}".format(self.clusters_count))
+        print("Top cluster tweets: \n{}".format(self.df_results.to_string()))
+
+
+def authorise_api():
+    """Authorise access to twitter API and return the api handler"""
+    load_dotenv()
+    consumer_key = os.getenv("CONSUMER_KEY")
+    consumer_key_secret = os.getenv("CONSUMER_KEY_SECRET")
+    access_token = os.getenv("ACCESS_TOKEN")
+    access_token_secret = os.getenv("ACCESS_TOKEN_SECRET")
+
+    auth = tw.OAuthHandler(consumer_key, consumer_key_secret)
+    auth.set_access_token(access_token, access_token_secret)
+    api = tw.API(auth, wait_on_rate_limit=True)
+
+    return api
+
+
+def get_top_trends(api):
+    """Returns a list of top trends at the specified location"""
+    # Current location: New Delhi, India. WOEID = 1 for global trending
+    latitude = 28.644800
+    longitude = 77.216721
+
+    locations = api.trends_closest(latitude, longitude)
+    woeid = locations[0]["woeid"]
+
+    trends = api.trends_place(woeid)
+    trends_dict = trends[0]["trends"]
+
+    return [trends_dict[0]]
+
+
+def de_emojify(text):
```

Comment:
make a class called `CleanData` and put all the cleaning methods inside that class.